src/SFA.DAS.EAS.Web/Views/EmployerCommitments/TransferDetails.cshtml

```diff
@@ -0,0 +1,120 @@
+﻿@using SFA.DAS.EmployerCommitments.Web
```

Comment:
Did Rajesh have any input into this page? It's big enough that he should create most of the markup, ideally, and in the very least have sight of it. I think some of the spacing/alignment looks out. Also things are capitalised that aren't in the prototype.

Reply:
He's looking into it now. I created a demo controller action for him to consume